github/github.go

```diff
@@ -681,6 +683,36 @@ func (r *ErrorResponse) Error() string {
 		r.Response.StatusCode, r.Message, r.Errors)
 }
 
+func (r *ErrorResponse) Is(target error) bool {
+	v, ok := target.(*ErrorResponse)
+	if !ok {
+		return false
+	}
+	if (r.Message != v.Message) || (r.DocumentationURL != v.DocumentationURL) {
+		return false
+	}
+	if r.Response != nil {
+		if r.Response.StatusCode != v.Response.StatusCode {
+			return false
+		}
+	}
+	if len(r.Errors) != len(v.Errors) {
+		return false
+	}
+	for idx := range r.Errors {
+		if r.Errors[idx] != v.Errors[idx] {
+			return false
+		}
+	}
+	if r.Block != nil {
+		if r.Block.Reason != v.Block.Reason || *(r.Block).CreatedAt !=
+			*(v.Block).CreatedAt {
+			return false
```

Comment:
Please add a unit test that covers this line.

Reply:
Done.